api/src/utils/apply-query.ts

```diff
@@ -606,132 +607,129 @@ export function applyFilter(
 				}
 			}
 
-			if (operator === '_eq') {
-				dbQuery[logical].where(selectionRaw, '=', compareValue);
-			}
+			let value = compareValue;
```

Comment:
I think this `value` should remain within the case scopes as it is a little ambiguous on the top level and only used in selected cases.

Reply:
Unfortunately there are multiple blocks using (defining) value and you can't redefine the same variable name within Switch. However I did go through the code and realised there isn't actually any need for a separate value variable so it is now removed.